R/assign_demographic_cohort.R

```diff
@@ -0,0 +1,108 @@
+#' Assign the demographic cohort variables
+#'
+#' @param data A data frame with the required variables for assignment
+#'
+#' @return A data frame with ten additional variables relating to the different cohorts
+#' @export
```

Comment:
Remove export to non top level functions

Reply:
Done